docs/UserGuide.md

```diff
@@ -1,10 +1,10 @@
 ---
-layout: page
-title: User Guide
+layout: page title: User Guide
 ---
 
 _Teaching Assistant_ is a desktop application that helps **JC/Secondary school teachers** have an efficient tool to keep
-track of their **schedules, contacts and tasks easily.** While it has a GUI, most of the user interactions rely on typing commands as user inputs.
+track of their **contacts and entries like tasks or schedules easily.** While it has a GUI, most of the user
+interactions rely on typing commands as user inputs.
```

Comment:
I wrote a small intro last time:

Welcome to the User Guide of **Teaching Assistant**!

Are you a JC/Secondary school teacher, having troubles keeping track of all your consultations, meetings and your students' contacts? No worries! Our application, **Teaching Assistant** will provide an all-in-one platform for you to organise your entries _(tasks and schedules)_ and contacts!

Teaching Assistant mainly uses a Command Line Interface (CLI). This means that a user can use the application by typing into a Command Box. For users who type fast, they can use this application more efficiently than other applications that heavily use Graphical User Interface (GUI), where a user uses the application by interacting with graphical features such as buttons.

If you are interested, jump to [Quick Start](#quick-start) to learn how to learn how to start using **Teaching Assistant**.